src/app/loginComponents/accounts/external/accounts.component.ts

```diff
@@ -238,4 +247,45 @@ export class AccountsExternalComponent implements OnInit, OnDestroy {
     this.ngUnsubscribe.next();
     this.ngUnsubscribe.complete();
   }
+
+  revokeToken() {
+    let dialogRef = this.dialog.open(RevokeTokenDialogComponent, {
+      width: '600px',
+      data: { revokeButtonClicked: this.revokeButtonClicked },
+    });
+    dialogRef.afterClosed().subscribe((buttonClicked) => {
+      if (buttonClicked) {
+        this.deleteToken(TokenSource.DOCKSTORE).subscribe(
+          () => {
+            this.revokeTokenSuccess();
+          },
+          (error: HttpErrorResponse) => {
+            console.log(error);
+            this.revokeTokenFailure();
+          }
+        );
+      }
+    });
+  }
+
+  /**
+   * What happens when revoking the Dockstore Token has succeeded.
+   *
+   * @private
```

Comment:
Method is not private (but it probably should be). We have this pattern all over, but it looks like they say you should generally not use it: https://typedoc.org/tags/private/

Please at least add `private` to the method declaration; optionally remove it. A few more instances of this are below.